fusion-plugin-react-router/__tests__/plugin.js

```diff
@@ -439,6 +458,29 @@ if (__BROWSER__) {
     const simulator = setup(app);
     await simulator.render('/');
   });
+  test('handles url with invalid URI encoding', async () => {
```

Comment:
is this a duplicate test?

Reply:
one is for server and one is for browser

Reply:
will rename them!